## Script Descriptions
* **addFileName2header.pl**	-	Add the name of the fasta/fastq file (without the extension) to the header. Useful when merging multiple fasta/fastq files and wish to easily keep track of sequences
* **batchBlast.pl**	-	Run multiple Blasts at once in an "embarassingly" parallel manner
* **calcN50.pl**	-	Calculate N50 and L50 values for a fasta file
* **chopper.pl**	-	Chop a file (fasta/fastq/tab-delimited/multiple-line) into multiple parts.
* **createFastq.pl**	-	Use the fasta and quality files to produce fastq files
* **CRISPR\_spacer\_extractor.pl**	-	Given a fasta file with repeat sequences and a contig fasta file, get the positions of these repeats in the contigs and find the coordinates of spacers
* **curateDB.pl**	
* **dereplicate.pl**	
* **extractSeqs.pl**	
* **extractSubSeq.pl**	
* **findStretchesOfNs.pl**	
* **gcSkew.pl**	
* **genomeCheck.pl**	
* **getRandomData.pl**	
* **iClust.pl**	
* **interleave.pl**	
* **kmerFreq.pl**	
* **length+GC.pl**	
* **limit2Length.pl**	
* **mapper.pl**	
* **parseFastq.pl**	
* **renameHeaders.pl**	
* **sangerSeqParser.pl**	
* **separateInterleaved.pl**	-	Separate interleaved files.
* **tetramer\_freqs\_esom.pl**	
* **toPhylipAndBack.pl**	
* **tinySeq2table.xslt**	-	restructure TinySeq format XML file to a *tab-delimited* file
* **tinySeq2fasta.xslt**	-	restructure TinySeq format XML file to a *FASTA* format file
* **triage.pl**	
* **U2T.pl**	
